--- awscli-1.16.278/setup.py.orig	2019-11-08 14:14:34.000000000 -0500
+++ awscli-1.16.278/setup.py	2019-11-11 06:37:15.867209395 -0500
@@ -39,14 +39,14 @@
     install_requires.append('PyYAML>=3.10,<=3.13')
 
     # Colorama removed support for EOL pythons.
-    install_requires.append('colorama>=0.2.5,<=0.3.9')
+    install_requires.append('colorama>=0.2.5')
 elif sys.version_info[:2] == (3, 3):
     install_requires.append('PyYAML>=3.10,<=3.13')
     # Colorama removed support for EOL pythons.
-    install_requires.append('colorama>=0.2.5,<=0.3.9')
+    install_requires.append('colorama>=0.2.5')
 else:
     install_requires.append('PyYAML>=3.10,<5.2')
-    install_requires.append('colorama>=0.2.5,<0.4.2')
+    install_requires.append('colorama>=0.2.5')
 
 
 setup_options = dict(
